apollo-router/src/configuration/tests.rs

```diff
@@ -678,3 +668,46 @@ fn visit_schema(path: &str, schema: &Value, errors: &mut Vec<String>) {
         _ => {}
     }
 }
+
+#[test]
+fn test_configuration_validate_and_sanitize() {
+    let conf = Configuration::builder()
+        .supergraph(Supergraph::builder().path("/g*").build())
+        .build()
+        .unwrap()
+        .validate()
+        .unwrap();
+    assert_eq!(&conf.supergraph.sanitized_path(), "/g:supergraph_route");
```

Comment:
what is the meaning of the `:supergraph_route` suffix?

Reply:
It's just the Axum notation for path parameter